apps/cowswap-frontend/src/modules/trade/pure/CompatibilityIssuesWarning/index.tsx

```diff
@@ -27,10 +27,11 @@ export const CompatibilityIssuesWarning = memo((props: CompatibilityIssuesWarnin
           <>
             <p>CoW Swap requires offline signatures, which is currently not supported by some wallets.</p>
             <p>
+              {/*TODO: I think FAQ doesn't exist anymore. Point to docs instead?*/}
```

Comment:
This banner is displayed only when wallet is not supported.
We detect a wallet as unsupported only when its name in:
```
// Smart contract wallets are filtered out by default, no need to add them to this list
const UNSUPPORTED_WC_WALLETS = new Set(['DeFi Wallet', 'WallETH'])
```

Probably those:
 - https://crypto.com/eea/onchain
 - https://walleth.org/

Reply:
We need to retest those wallets, because this code is super old, probably we can just delete this banner and logic